eeps/eep-0056.md

```diff
@@ -1,6 +1,6 @@
     Author: Maria Scott <maria-12648430(at)hnc-agency(dot)org>,
             Jan Uhlig <juhlig(at)hnc-agency(dot)org>
-    Status: Final/OTP-24 Implemented in OTP 24
+    Status: Final/24.0 Implemented in OTP release 24.0
     Type: Standards Track
     Created: 04-Mar-2021
     Erlang-Version: 24.0
```

Comment:
I'm not able to suggest the Post-History here.
```
    Post-History: 08-Mar-2021, 17-Mar-2021, 23-Mar-2021, 31-Mar-2021, 06-Sep-2023
```

Reply:
This was discussed in PR erlang/otp#4521 before this EEP was created.